config/dev/openstack-managedcluster.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: ManagedCluster
+metadata:
+  name: openstack-dev
+  namespace: ${NAMESPACE}
+spec:
+  template: openstack-standalone-cp-0-0-1
+  credential: openstack-cluster-identity-cred
+  config:
+    controlPlaneNumber: 1
+    workersNumber: 1
+    region: RegionOne
```

Comment:
It does not match the configuration defined in the `openstack-standalone-cp` helm chart (here we have `region` but in the cluster template helm chart we have `clusterIdentity.region` https://github.com/Mirantis/hmc/pull/696/files). 

Anyway, I suggest moving this OpenStack ManagedCluster testing YAML to a separate PR. Currently, we lack both an OpenStack standalone cluster template and the required credentials handling. It isn’t critical, but I prefer patches to focus only on related changes for better clarity and organization.

Reply:
so should I move this file to the cluster template PR?